spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/sql/R2dbcCloudSqlEnvironmentPostProcessorTests.java

```diff
@@ -100,18 +103,44 @@ void testSetR2dbcProperty_mySql_urlProvidedByUserIgnored() {
   }
 
   @Test
-  void testSetR2dbcProperty_postgres_defaultUsername() {
+  void testSetR2dbcProperty_postgres() {
+    validateDatabaseProperties(new String[] {"io.r2dbc.postgresql"},
+        "postgres",
+        "r2dbc:gcp:postgres://my-project:region:my-instance/my-database");
+  }
+
+  @Test
+  void testSetR2dbcProperty_mysql() {
+    validateDatabaseProperties(new String[] {"dev.miku.r2dbc.mysql"},
+        "root",
+        "r2dbc:gcp:mysql://my-project:region:my-instance/my-database");
+  }
+
+  /**
+   * Accepts a list of packages to _keep_, and tests that correct properties for the database type
+   * got injected into context.
+   * The input is used to calculate the inverse list of packages to _remove_ that is passed to
+   * {@code FilteredClassLoader}
```

Comment:
Very helpful documentation! Very much optional but is it possible to move this line to an inline comment inside the method and perhaps add a line in the javadoc about what the method is verifying?

Reply:
done -- I think I've made the description clearer. Take a look?

Reply:
Looks great! Thank you